solana/program_v2/programs/gateway_v2/src/state/gatekeeper.rs

```diff
@@ -269,7 +281,7 @@ impl OnChainSize for GatekeeperFees {
 bitflags! {
      /// The flags for a key on a gatekeeper
      #[derive(AnchorSerialize, AnchorDeserialize)]
-     pub struct GatekeeperKeyFlags: u16{
+     pub struct GatekeeperKeyFlags: u32{
```

Comment:
Does adding one more flag mean it must increase to u32? I don't remember the size constraints, just thought I'd ask

Reply:
@DiosAkhilleus it reached the limit of 16 bits (i.e. 16 flags) so we had to increase the size to accomodate more